### PR TITLE
  Expose cors headers so clients subject to it can access it.

### PR DESCRIPTION
**Pull request by Natim**
Replace https://github.com/mozilla-services/readinglist/pull/129

### CODE REVIEW

File: cliquet/tests/testappviews.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from cliquet.resource import BaseResource, ResourceSchema, crud
```
Comment (ametaireau) [inline cliquet/tests/testappviews.py]: I believe we should put that in a separate folder, exactly like we would do for a real app.

File: cliquet/tests/testappviews.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import colander
+
+
+class MushroomSchema(ResourceSchema):
+    name = colander.SchemaNode(colander.String())
+
+
+@crud()
+class Mushroom(BaseResource):
+    mapping = MushroomSchema()
```
Comment (leplatrem) [inline cliquet/tests/testappviews.py]: Couldn't we factorize with this ? https://github.com/mozilla-services/cliquet/blob/exposed-cors-headers/cliquet/tests/resource/test_views.py#L16-L22

File: cliquet/tests/test_cors.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+from .support import BaseWebTest, unittest
+
+
+def get_available_headers(headers):
+    access_control_headers = headers['Access-Control-Expose-Headers']
+    return [x.strip() for x in access_control_headers.split(',')]
+
+
+class CORSTest(BaseWebTest, unittest.TestCase):
+    def test_preflight_headers_for_default(self):
```
Comment (ametaireau) [inline cliquet/tests/test_cors.py]: Use sentences in your functions.

**Comment by Natim** [comment]
Updated.

**Comment by Natim** [comment]
It fails with py34

File: cliquet/tests/test_cors.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import json
+from .support import BaseWebTest, unittest
+
+
+def get_available_headers(headers):
```
Comment (ametaireau) [inline cliquet/tests/test_cors.py]: get_exposed_headers

**Pull request merged**